.github/workflows/publishWheelRelease.yml

```diff
@@ -0,0 +1,129 @@
+# Copyright 2023 The StableHLO Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Python Wheel Release
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+# Ensure that only a single job or workflow using the same
+# concurrency group will run at a time. This would cancel
+# any in-progress jobs in the same github workflow and github
+# ref (e.g. refs/heads/main or refs/pull/<pr_number>/merge).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cmake-build:
+    # Only run scheduled CI on main repo
+    if: (github.repository == 'openxla/stablehlo' || github.event_name != 'schedule')
+    name: "cmake-build ${{ github.event_name == 'schedule' && '(llvm-project@HEAD)' || ''}}"
+    env:
+      LLVM_PROJECT_DIR: "llvm-project"
+      LLVM_BUILD_DIR: "llvm-build"
+      # The setup.py script expects the StableHLO build to be in 'build'
+      STABLEHLO_BUILD_DIR: "build"
```

Comment:
The fact the build directory has to be called build since I reference it in setup.py is cursed.